packages/sit-onyx/src/types/forms.ts

```diff
@@ -0,0 +1,48 @@
+import type { DensityProp } from "../composables/density";
+import type { TruncationType } from "./fonts";
+
+// IMPORTANT: When changing this type, make sure to implement the feature
+// for the following components:
+// - Checkbox / Checkbox group
+// - Radio button / Radio button group
+// - Select
+// - Listbox
+// - ...
+
+/**
+ * Select option that is e.g. used inside the select, radio button / checkbox group.
+ */
+export type SelectOption<TValue extends SelectOptionValue = SelectOptionValue> = DensityProp & {
+  /**
+   * Value of the option when it is selected.
+   */
+  value: TValue;
```

Comment:
Please add comment (or rename) that this is the unique identifier for the option in the list